tests/MercadoPago/Client/Integration/Order/OrderClientITTest.php

```diff
@@ -114,31 +122,128 @@ private function createRequestCapture(): array
+            $order = $client->create($request, $request_options);
+            $this->assertNotNull($order->id);
+
+            sleep(3); // sleep to avoid error 422 when create and cancel ocurrs just in time
+
+            $order_cancelled = $client->cancel($order->id, $request_options);
+            $this->assertNotNull($order_cancelled->id);
+            $this->assertSame("cancelled", $order_cancelled->status);
+
         } catch (MPApiException $e) {
             $apiResponse = $e->getApiResponse();
             $statusCode = $apiResponse->getStatusCode();
             $responseBody = json_encode($apiResponse->getContent());
             $this->fail("API Exception: " . $statusCode . " - " . $responseBody);
+
         } catch (\Exception $e) {
             $this->fail("Exception: " . $e->getMessage());
         }
     }
+
+    public function testProcessSuccess(): void
+    {
+        try {
+            $client = new OrderClient();
+            $request = $this->createRequestProcess();
+            $request_options = new RequestOptions();
+            $request_options->setCustomHeaders([ "X-Sandbox: true"]);
+            $order = $client->create($request, $request_options);
+            $this->assertNotNull($order->id);
+
+            $order_processed = $client->process($order->id, $request_options);
+            $this->assertSame("processed", $order_processed->status);
+
+        } catch (MPApiException $e) {
+            $apiResponse = $e->getApiResponse();
+            $statusCode = $apiResponse->getStatusCode();
+            $responseBody = json_encode($apiResponse->getContent());
+            $this->fail("API Exception: " . $statusCode . " - " . $responseBody);
+
+        } catch (\Exception $e) {
+            $this->fail("Exception: " . $e->getMessage());
+        }
+    }
+
+    private function createRequestProcess(): array
+    {
+        $client_token = new CardTokenClient();
+        $card_token = $client_token->create($this->createCardTokenRequest());
+        $this->assertNotNull($card_token->id);
+
+        $request = [
+            "type" => "online",
+            "processing_mode" => "manual",
+            "total_amount" => "200.00",
+            "external_reference" => "ext_ref_1234",
+            "transactions" => [
+                "payments" => [
+                    [
+                        "amount" => "200.00",
+                        "payment_method" => [
+                            "id" => "master",
+                            "type" => "credit_card",
+                            "token" => $card_token->id,
+                            "installments" => 1,
+                        ]
+                    ]
+                ]
+            ],
+            "payer" => [
+                "email" => "test_1731350184@testuser.com",
+            ]
+        ];
+        return $request;
+    }
+
+    private function createCardTokenRequest(): array
+    {
+        $request = [
```

Comment:
Veja se funciona Dani voltar direto o array sem necessitar de uma variavel, fazer na requets de order tb
```suggestion
        return [
```

Reply:
tipo, fazer tudo dentro da função de teste??
isso não iria replicar código?